test/test_transformers.py

```diff
@@ -1627,8 +1638,8 @@ def test_mem_efficient_attention_vs_math_ref_grads(self, batch_size: int, seq_le
 
         # Create real output
         with sdp_kernel(enable_mem_efficient=True, enable_flash=False, enable_math=False):
-            # See check_gpu_sm86_head_dim_128 in pytorch/aten/src/ATen/native/transformers/cuda/sdp_utils.h
-            if isSM86Device and head_dim == 128:
+            # See check_head_dim_gt64_and_sm_ge86 in pytorch/aten/src/ATen/native/transformers/cuda/sdp_utils.h
+            if (isSM86Device or isSM89Device) and (head_dim > 64 and head_dim <= 128):
```

Comment:
I know range(64, 129) is a huge overkill there, but it looks syntactically pleasing, isn't it?
```suggestion
            if isSM86or89Device and head_dim in range(65, 129):
```